src/users/routes/usersRouter.js

```diff
@@ -1,20 +1,18 @@
 import { Router } from 'express';
-
 import { userController } from '../controllers';
-import verifyTokken from '../middlewares/verifyToken';
 
 const router = Router();
 
-router.get('/', verifyTokken, userController.readAll);
```

Comment:
why user controller functions do not require validation? all controllers should be alike